pkg/ambassador/tap/conduit/conduit.go

```diff
@@ -174,7 +172,7 @@ func (c *Conduit) GetConduit() *ambassadorAPI.Conduit {
 // GetStreams returns the local IPs for this conduit
 func (c *Conduit) GetIPs() []string {
 	if c.connection != nil {
-		return c.connection.GetContext().GetIpContext().GetSrcIpAddrs()
```

Comment:
If it wasn't for conduit_test this function wouldn't be public, as it accesses the connection and thus is expected to be invoked with held mu lock. Currently not a problem, but might be in the future...